### ISSUE TITLE
Add Murder Day "Last Request"

### ISSUE THREAD

**Issue by mrnoahfink** [Scags/TF2-Jailbreak-Redux#137]
As the title said. Are you able to add Murder Day for the last request?

**Comment by Scags** [comment]
No. Unfortunately I don't have the time due to school and other projects.

**Comment by mrnoahfink** [comment]
No need rush tho. I was going to use this https://forums.alliedmods.net/showthread.php?p=2519830 but I realize that doesn't have an enable and disable command.

Also, could you add a warden vote system? Red can choose any blu player to become warden.

**Comment by mrnoahfink** [comment]
also can guard have time or how long guard ban need?

**Comment by mrnoahfink** [comment]
I wonder if you are able to create another menu like "Last Request" But Death Request after the 
last red team picked for LR. 
![deathrequest](https://user-images.githubusercontent.com/66857858/104972721-6abb4380-59a7-11eb-9674-4ae02fe85c73.png)
here example ^

Also is there cvar to disable LR and/or Death Request?

Thanks!
